mws/mws.py

```diff
@@ -135,7 +135,10 @@ class DictWrapper(object):
 
     def __init__(self, xml, rootkey=None):
         if isinstance(xml, bytes):
-            xml = xml.decode()
+            try:
+                xml = xml.decode()
+            except:
+                xml = xml
```

Comment:
Should we not be more explicit in our exception handling here? I think the biggest issue is the `UnicodeDecodeError` ?

Reply:
Sorry, I must have pushed this by mistake. I just had an error and needed to get rid of that.

Reply:
No problem, I know this has caused an issue for another user since the `.decode()` has been implemented, so its not a bad think to have, however,  global exception trapping might not be best.

I made a rough outline of the issue on #188 (poorly worded and on my mobile late at night!) so this does solve that particular issue if its reverting and only on the specific decode error.